ua/org.eclipse.help.base/build.properties

```diff
@@ -11,9 +11,10 @@
 # Contributors:
 #     IBM Corporation - initial API and implementation
 ###############################################################################
-source..=src/,\
-         src_demo/
-src.includes=schema/,about.html
+source.. = src/,\
```

Comment:
`o.e.help.base` is already provided by https://github.com/eclipse-platform/eclipse.platform/pull/862

Reply:
That's true, this one supercedes #862, I forgot to mention that.
What kind of changes do you expect for this PR?

Reply:
No need to supersede, one can build upon it and acknowledge my contribution too... small as it is.

And I thank you for your input in my PR too, specially for [pointing out the missing entry](https://github.com/eclipse-platform/eclipse.platform/pull/862#issuecomment-1820673386) :-)